mantidproject/mantid
# Visual display of fit benchmarking results

The fit benchmark now produces plots of the initial guess and best result along with the raw data. 

**To test:**
Run fit benchmarking and check the plots that are produced correctly (in MantidPlot directory in build).



Fixes #18173. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
